src/citrine/resources/gemd_resource.py

```diff
@@ -0,0 +1,268 @@
+"""Collection class for generic GEMD objects and templates."""
+from collections import defaultdict
+from typing import Type, Union, Optional, List, Tuple
+from uuid import UUID
+
+from gemd.util import writable_sort_order
+from gemd.entity.base_entity import BaseEntity
+from gemd.entity.link_by_uid import LinkByUID
+
+from citrine._utils.functions import format_escaped_url
+from citrine.resources.api_error import ApiError
+from citrine.resources.data_concepts import DataConcepts, DataConceptsCollection
+from citrine.resources.delete import _async_gemd_batch_delete, _poll_for_async_batch_delete_result
+from citrine._session import Session
+
+
+class GEMDResourceCollection(DataConceptsCollection[DataConcepts]):
+    """A collection of any kind of GEMD objects/templates."""
+
+    _path_template = 'projects/{project_id}/storables'
+    _dataset_agnostic_path_template = 'projects/{project_id}/storables'
+    _individual_key = None
+    _collection_key = None
+    _resource = DataConcepts
```

Comment:
Why are these defined in this class?  The unit tests pass with these commented out.

Reply:
Yes they appear unused, so am removing in the changes.

Reply:
They're not used in this case, but they are part of the interface, so I would vote for retaining them.

But I won't block on it, and am approving.